about/get-involved.md

```diff
@@ -0,0 +1,94 @@
+
+## Support other users ## {#support-users}
+Many users come and ask questions about using Keyman in our [online community](https://community.software.sil.org/c/keyman).
+Each question that you answer helps many other users who may be searching for similar answers, and allows the developers to focus
+on improving Keyman! And if you have new knowledge about Keyman, you could even contribute that to the documentation &mdash; keep reading.
+
+* [Help other users in the Keyman Community forums](https://community.software.sil.org/c/keyman)
+* [Ask and answer technical questions about Keyman on Stack Overflow](https://stackoverflow.com/questions/tagged/keyman)
+
+## Test a new version ## {#test-beta}
+We frequently release updates to Keyman. Independent testing and reporting of issues is greatly appreciated. When
+you test bug fixes and report issues, you help those who are depending on Keyman to work in their language, many
+of whom do not have the capacity to test new versions themselves.
+
+* [Download a pre-release version](/downloads/pre-release)
+* [Report issues and feature requests](https://github.com/keymanapp/keyman/issues/new/choose)
+
+## Write some code ## {#write-code}
+Because Keyman is open source, anyone can contribute code. We love pull requests, forks and code reviews ;-). Keyman
+includes C++, Java, Swift,ObjC, Delphi, Javascript and Python code, so there is something for every developer!
+[Explore our primary GitHub repo](https://github.com/keymanapp/keyman) and find out more!
+
+* [Keyman on GitHub](https://github.com/keymanapp/keyman)
+* [Work on issues we need help with](https://github.com/keymanapp/keyman/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
+  <!-- TODO: Add more introductory text to readme.md on keymanapp/keyman -->
+
+## Improve the documentation ## {#improve-docs}
+Keyman has a lot of documentation! While we work hard to keep Keyman documentation up to date, there is
+always room for improvement. If you spot a gap in the documentation, or a mistake, you can submit an update
+through our GitHub documentation repository, by editing the source files directly online in GitHub!
+
+* [Read the documentation](https://help.keyman.com/)
+* [Start exploring the documentation repository](https://github.com/keymanapp/help.keyman.com)
+  <!-- TODO: update help.keyman.com readme to clarify how users can edit files? -->
+
+## Join the team! ## {#join-team}
+Do you have a passion for software development? Would you be interested in joining SIL and working full-time on
+developing software that serves the needs of language communities around the world?
+[Write us](https://software.sil.org/about/contact/), or ping us on [@Keyman](https://twitter.com/keyman) and
+introduce yourself &mdash; we'd love to meet with you!
+
+* [Write to us](https://software.sil.org/about/contact/)
+* [Send us a tweet or DM on Twitter](https://twitter.com/keyman)
+
+## Test Links (not for production)
```

Comment:
I tried to incorporate all your existing comments. Here's a test section to see how the Keyman hosts get munged.